src/pages/Main.js

```diff
@@ -1,66 +1,137 @@
 import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
-
-const day = ['일', '월', '화', '수', '목', '금', '토'];
-const Main = () => {
-  const [startDescription, setStartDescription] =
-    useState(' 오늘도 상쾌하게 시작해볼까요?');
-  const [time, setTime] = useState(new Date().getMinutes());
+import Modal from '../components/Modal/Modal';
+import SendTimeModal from '../pages/Main/SendTimeModal/SendTimeModal';
+
+export default function Main() {
+  const [time, setTime] = useState({
```

Comment:
`Main.js`파일이  `Routes.js`파일 안에 import되어있다면, useHistory 사용안하시고 바로 Main 컴포넌트에서 props로 내려서 history객체 사용하실 수 있습니다! 위의 `import{ useHistory } from 'react-router-dom'` 삭제해주시고 아래처럼 사용하시면 더 좋을 듯합니다!

```suggestion
export default function Main({history}) {
  const [time, setTime] = useState({
```

Reply:
오호~!! 감사합니다~!!